README.md

```diff
@@ -179,6 +179,10 @@ end
   - Called from controller actions or `before_action`s
   - Raises `Moat::NotAuthorizedError` if `user` is not permitted to take `action` according to `policy`
   - Automagically looks up policy if not given
+- `authorized?(resource, action = action_name, user: moat_user, policy: <optional>)`
+  - Called in controller methods
+  - Returns `true` if `user` is permitted to take `action` on the resource according to `policy`, otherwise it return `false`
```

Comment:
1. Grammar error, "return"
2. It looks like the documentation for `authorization` above should match this.